py/bodylineleft.py

```diff
@@ -1,19 +1,90 @@
+    with table(cls="height") as bodylineleft:
+        with tr():
+            #Creates vertical line
+            td(cls="td-valign left-line-pad").add(div(cls="vertical-left"))
+            #Makes table for entries.
+            with td(cls="td-valign left-pad").add(table()):
+                with tr().add(td(cls="td-valign")).add(div(cls="title-div")):
+                    #Makes section title and horizontal line.
+                    if isinstance(title, str):
+                        h1(title, cls="section-title")
+                        hr(cls="horiz")
+                    else:
+                        raise TypeError("Title must be a string.")
+                for i in range(len(entries)):
+                    entry = entries[i]
+                    #Adjusts padding for the first entry of the section.
+                    entry_class = "td-valign entry"
+                    if i == 0:
+                        entry_class = "td_valign first-entry"
+                    if isinstance(entry, Entry):
+                        #Creates table for the entry.
+                        with tr().add(td(cls=entry_class)).add(table()):
+                            tr().add(td(cls="td-valign")).add(h2(entry.title, cls="title"))
+                            #Adds details section if the entry has one.
+                            if entry.details != None:
+                                with tr().add(td(cls="td-valign detail-pad")):
+                                    #Iterates through the details array and generates HTML depending on object type.
+                                    for line in entry.details:
+                                        if isinstance(line, Link):
+                                            with p(cls="details").add(u()):
+                                                a(line.text, cls="link", href=line.url, target="_blank")
+                                        elif isinstance(line, Linebreak):
+                                            for i in range(line.numBreaks):
+                                                br()
+                                        elif isinstance(line, Text):
+                                            p(line.text, cls="details")
+                                        else:
+                                            raise TypeError("Elements of details must be either a Text, Linebreak or Link object.")
+                            #Iterates through the body of the entry to create HTML for rendering a block of content or an image.
+                            for elem in entry.body:
+                                if isinstance(elem, Content):
+                                    with tr().add(td(cls="td-valign left-desc-pad")):
+                                        for line in elem.desc:
+                                            if isinstance(line, Link):
+                                                with p(cls="description").add(u()):
```

Comment:
Was there a reason you used a <u> tag to underline the link instead of just styling the <a> tag to have an underline in .css?

Reply:
Using css underline wasn't being displayed on one of the email platforms.